perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ConsentsManagerBlImpl.java

```diff
@@ -29,4 +36,14 @@ public PerunBl getPerunBl() {
 	public void setPerunBl(PerunBl perunBl) {
 		this.perunBl = perunBl;
 	}
+
+	@Override
+	public void addFacility(PerunSession sess, ConsentHub consentHub, Facility facility) throws RelationExistsException {
+		consentsManagerImpl.addFacility(sess, consentHub, facility);
+	}
+
+	@Override
+	public void removeFacility(PerunSession sess, ConsentHub consentHub, Facility facility) throws RelationNotExistsException {
+		consentsManagerImpl.removeFacility(sess, consentHub, facility);
```

Comment:
we will want to remove whole consentHub if we removed last facility. it should replace current solution in #3528 when removing facility